day-to-day.md

```diff
@@ -129,21 +135,12 @@ Definition of a good meeting:
 
 The entire team meets once a week for lunch (paid for by Base Two) to go over the state of the company. The meeting typically lasts about 1.5 hours. The agenda for each meeting is prepared that morning (in a shared Google Doc) and all team members are welcome to contribute. We cover any combination of the following (incomplete) list of topics:
 
-- Financials
-	- Account balances
-	- Invoices/billing
-	- Profitability
-	- Hours
+- Financials - Account balances - Invoices/billing - Profitability - Hours
```

Comment:
It looks like these (and two groupings below) sub-bullets got mangled in the re-format.